channel_dns.py

```diff
@@ -278,10 +279,12 @@ def format_additional_data(self, **kwargs):
                     .format(filename=kwargs['src_data']['dtrace_filename'])
 
             if 'windows_desktopini_access_username' in kwargs['src_data']\
-               and 'windows_desktopini_access_domain' in kwargs['src_data']:
-                additional_report += '\nWindows Directory Browsing By: {domain}\{username}'\
+               and 'windows_desktopini_access_domain' in kwargs['src_data']
+               and 'windows_desktopini_access_hostname' in kwargs['src_data']:
+                additional_report += '\nWindows Directory Browsing By: {domain}\{username} from {hostname}'\
                     .format(username=kwargs['src_data']['windows_desktopini_access_username'],
-                            domain=kwargs['src_data']['windows_desktopini_access_domain'])
+                            domain=kwargs['src_data']['windows_desktopini_access_domain'],
+                            hostname=kwargs['src_data']['windows_desktopini_access_hostname'])
```

Comment:
perhaps better to separate the if statement (again for backward compat) into
```python
if 'windows_desktopini_access_username' in kwargs['src_data']\
  and 'windows_desktopini_access_domain' in kwargs['src_data']:
     if 'windows_desktopini_access_hostname' in kwargs['src_data']:
            additional_report += '\nWindows Directory Browsing By: {domain}\{username} from {hostname}'\
                       .format(username=kwargs['src_data']['windows_desktopini_access_username'],
                                    domain=kwargs['src_data']['windows_desktopini_access_domain'],
                                    hostname=kwargs['src_data']['windows_desktopini_access_hostname'])
       else:
              additional_report += '\nWindows Directory Browsing By: {domain}\{username}'\
                       .format(username=kwargs['src_data']['windows_desktopini_access_username'],
                                    domain=kwargs['src_data']['windows_desktopini_access_domain'])
```

Reply:
Also fixed.